docs/essentials/skip-building-some-commits-with-ci-skip.md

```diff
@@ -7,3 +7,9 @@ description: If you'd like your commit, or a series of commits that you're pushi
 If you'd like your commit, or a series of commits that you're pushing, to not
 trigger a build, just write `[ci skip]` or `[skip ci]` somewhere in your 
 commit's message.
+
+!!! info "Usage with squashed commits"
+	When squashing commits before merging in another branch all commit messages
+will be merged as well. If a previous commit in the current branch contains
+`[ci skip]` or `[skip ci]` it will be propagated and the merged commit wil
```

Comment:
`wil` -> `will`

Reply:
Thanks @radwo there's also a problem with the formatting.